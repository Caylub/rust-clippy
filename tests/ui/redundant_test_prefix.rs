#![warn(clippy::redundant_test_prefix)]

#[cfg(test)]
mod test {
    use super::*;

    #[test]
    fn test_redundant_name() {
        //
    }
}
