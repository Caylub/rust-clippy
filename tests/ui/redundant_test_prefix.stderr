error: this test is prefixed redundantly with `test`
  --> tests/ui/redundant_test_prefix.rs:8:5
   |
LL | /     fn test_redundant_name() {
LL | |         //
LL | |     }
   | |_____^
   |
   = help: consider removing the redundant `test` prefix from the test name
   = note: `-D clippy::redundant-test-prefix` implied by `-D warnings`
   = help: to override `-D warnings` add `#[allow(clippy::redundant_test_prefix)]`

error: aborting due to 1 previous error

